### ISSUE TITLE
Windows fd issue

### ISSUE THREAD

**Issue by HrMitrev** [Yohannfra/Vim-Goto-Header#5]
Hi!
I like the plugin very much, but when using it under Windows, I got error because fd is not registered correctly. 

system('which.fd')[0] returns `C`

**Comment by Yohannfra** [comment]
Hi,

As far as I know the which command is only available on GNU/linux os so I can understand why it doesn't work on Windows.

I don't have any windows laptop available to try but I'll accept any patch for this issue if you can fix it.

**Comment by HrMitrev** [comment]
> Hi,
> 
> As far as I know the which command is only available on GNU/linux os so I can understand why it doesn't work on Windows.
> 
> I don't have any windows laptop available to try but I'll accept any patch for this issue if you can fix it.

You are right, but if you setup [scoop](https://scoop.sh/) you can install packages which add  `which.exe` to $PATH.
I am not experienced with vim syntax, but I can try to do a PR. 
One question about the code -  how could you add OR statement. Will it be something like:
```
let fdpath = system('which fd')[0]
if (fdpath ==# '/' || fdpath ==#'C')
```

**Comment by Yohannfra** [comment]
Yes this is the correct syntax. You can find a very helpful cheatsheet for vimscript here https://devhints.io/vimscript
